packages/ihe-gateway/ihe-gateway-setup.xml

```diff
@@ -6909,163 +6075,315 @@
+hvcNAQEBBQADggEPADCCAQoCggEBAMI6ckvXucEhND5kzHnUyqlBKjXc7YUSUYbkQiyDOHgwoiVJ&#xd;
+ba5qYJI6VdLKOsblCadSs2QBQe7C65m+xEMxxIP+6Qh3sB6KOhtormCyJoL/+Ra9TMlSOrp/0CU+&#xd;
+lU5hKbJa6qcIieoiMSeWLLh/u3GdnxZEwBU/DQQOJVN2sYr93EOydp+DWrHhjY64dm3R/WxIwCqW&#xd;
+4G4fwrfBKzDTrqav7zWuzVd8zSfZ3aNZwz8Zv4DepRrj18WMiU23agB+PDBY932veQOWWRR5VRYA&#xd;
+g5/frJ3Vn5EVJN1GNiReWGr7mef4vFSC1ctEc1A7DWdpZW1mRKFlxMM2zZlg0j96vekCAwEAAaOC&#xd;
+BEMwggQ/MIIDBAYDVR0jBIIC+zCCAveAggLzMIIC7zCCAdegAwIBAgIBATANBgkqhkiG9w0BAQsF&#xd;
+ADAuMSwwKgYDVQQDDCNNaXJ0aCBDb25uZWN0IENlcnRpZmljYXRlIEF1dGhvcml0eTAgFw0yMzEx&#xd;
+MjUwMTI1NDlaGA8yMDczMTEyNTAxMjU0OVowLjEsMCoGA1UEAwwjTWlydGggQ29ubmVjdCBDZXJ0&#xd;
+aWZpY2F0ZSBBdXRob3JpdHkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQC1emWrO7po&#xd;
+R8ALwyaYGf09T+tvPPIxQgov0jCNgxXtatln6uuK4HG6MJq3P+MXBRO8RO7XLDEi0ZXMoWa4p2iL&#xd;
++1I4NfwrtVsViEFUmmlKGd4gfoXrXETe58kaMc1zyyP4WjtGlkvC1HqjNhorgVpoWbgYTAlT0/Ci&#xd;
+sO8akO6D8Ad3cDdSg8lCXKG8KxTEQ3DeLPmuJ/PAXbDBJIsnQNq/UM4LzscFaS0Jjb0DHv78qXMo&#xd;
+RF2HTDCKDFfTJPG7GSIrf1JZ00etzj/JFgL+LngACF7Mtr+Q62NoxnafuuTtzffJn/OJyclq51EX&#xd;
+XCiRCImOdh5XPeStffJnFSU5dM1DAgMBAAGjFjAUMBIGA1UdEwEB/wQIMAYBAf8CAQAwDQYJKoZI&#xd;
+hvcNAQELBQADggEBAAN2VpLKfTppXzOjn/BQRbPgMqkRCM6kLYD9PWHW6Zv++5LIJTV7ctyrGpU+&#xd;
+CAs7zxZIf0oM3JggtkUw4V9zH6UVM9sX5HGIcaFu+5WRNUJaA0Iq/nxdD8Paa46cWgz59vVfvP6a&#xd;
+hOdapdtHfPRpzXN486f9xjpC5h8y27HoYuYrfIakSJUknrEHRg4/Q11to5ZM6EoejUa0Z9bB850h&#xd;
+/Z2IBQBihTTE/sVIbaTSzm9p2ssL00MUiABh95dVzGNp9+Op0AxhD5TAs1mm7o+ZcWM7rJWDcY8u&#xd;
+fApEbFiaqd5f3zDUui1kPuQOqBZvrqSzOIn5StiiATnyOx1POWgKtRswggEzBgNVHQ4EggEqBIIB&#xd;
+JjCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMI6ckvXucEhND5kzHnUyqlBKjXc7YUS&#xd;
+UYbkQiyDOHgwoiVJba5qYJI6VdLKOsblCadSs2QBQe7C65m+xEMxxIP+6Qh3sB6KOhtormCyJoL/&#xd;
++Ra9TMlSOrp/0CU+lU5hKbJa6qcIieoiMSeWLLh/u3GdnxZEwBU/DQQOJVN2sYr93EOydp+DWrHh&#xd;
+jY64dm3R/WxIwCqW4G4fwrfBKzDTrqav7zWuzVd8zSfZ3aNZwz8Zv4DepRrj18WMiU23agB+PDBY&#xd;
+932veQOWWRR5VRYAg5/frJ3Vn5EVJN1GNiReWGr7mef4vFSC1ctEc1A7DWdpZW1mRKFlxMM2zZlg&#xd;
+0j96vekCAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAKs19m/Zq+xt6uqWcW4knFFWW3C0T+tfpLBL7&#xd;
+Hx6DdtXiiZnzTyEcCQ1tvGyW2nWIr1k15fcr0ttn8itOritaWpsfuq+KnvyRbr//LQwNjkR+MHww&#xd;
+i7mXcDyDMbnEQKL/0QfBh6AoCFbOCklIEvhRziYpWdRvi8qHmUHZnNUZwDuTWEAtgZPaWvpOe3o4&#xd;
+NdgjX/YmjmRjRy3ivIHWy3C0LN6A/zzynTkdzQcm9H9084Lou5lWbJQhmi6SMpttk1FbUm/1xXbS&#xd;
+a9HzqieFDr1imv53n6Yz7MhUyYW+4I9J0VSzDOSqCLAshNsnvvdy3HbTOtRN6mzuFPCa+YJDn0EH&#xd;
+LPCdNE5936wZuQdQyBh9ifWJD1k8&#xd;
 </property>
-        <property name="KeyStorePassword">6da5e743-0ce6-4ce6-bd98-7826e90a066c</property>
-        <property name="TrustStore">/u3+7QAAAAIAAAAAkpaswKwSJn93N+dz8RVN6SRHN4Y=&#xd;
+        <property name="KeyStorePassword">dd0d4b52-a179-4392-8ba6-ef5544410bab</property>
+        <property name="TrustStore">/u3+7QAAAAIAAAADAAAAAgAgaWhlYXBpdGVzdC5pbm5vdmFyaGVhbHRoY2FyZS5jb20AAAGMEgOA&#xd;
+DwAFWC41MDkAAAVEMIIFQDCCBCigAwIBAgIIbOfwDBfNtM0wDQYJKoZIhvcNAQELBQAwgasxCzAJ&#xd;
```

Comment:
again, this shouldn't be here?

can you please verify that there is nothing aside from the Java Truststore in the top certificates section in the SSL manager